DEVELOPER.md

```diff
@@ -67,7 +67,9 @@ Prerequisites:
   * Install MariaDB command line client `mariadb`: 
     * deb: `sudo apt install -y mariadb-client`
     * rpm: `sudo dnf install -y mariadb`
-  * Install `sqlx-cli`: `cargo binstall sqlx-cli -y`
+  <!-- Installation using `cargo install` (rather than `cargo binstall`) is used intentionally -->
+  <!-- Details: https://github.com/launchbadge/sqlx/issues/3396 -->
```

Comment:
Can you create an issue in the upstream (`cargo-quickinstall`) repo so that this will eventually be fixed and we could revert back to binstall?

Also does this affect our CI?

Reply:
@sergiimk 

> Can you create an issue in the upstream (cargo-quickinstall) repo so that this will eventually be fixed and we could revert back to binstall?

yes, in my personal backlog

> Also does this affect our CI?

nope, it affects only new machine setup expertise